classes/image_types_fsl.bbclass

```diff
@@ -40,5 +40,5 @@ do_image_wic[depends] += " \
 IMAGE_CMD:wic:append:mxs-generic-bsp() {
 	# Change partition type for mxs processor family
 	bbnote "Setting partition type to 0x53 as required for mxs' SoC family."
-	echo -n S | dd of=$out${IMAGE_NAME_SUFFIX}.wic bs=1 count=1 seek=450 conv=notrunc
+	echo -n S | dd of=$out.wic bs=1 count=1 seek=450 conv=notrunc
```

Comment:
While I do not use this image_types class, I guess this should be:
s/$out.wic/${out}.wic/

Reply:
good catch! not testing is always very expensive.
Thanks